ctapipe/core/component.py

```diff
@@ -202,7 +202,7 @@ def from_name(cls, name, config=None, parent=None, **kwargs):
     def non_abstract_subclasses(cls):
         """
         get dict{name: cls} of non abstract subclasses,
-        subclasses can possibly be definded in plugins
+        subclasses can possibly be defined in plugins
```

Comment:
I am not sure what this line wants to say: Do you mean to say "It is possible to define subclasses in plugins"? Or "A subclass might have been defined in a plugin"? 

Is the intention to say that this function will examine plugins for subclasses too?

Reply:
> Is the intention to say that this function will examine plugins for subclasses too?

Yes that was the intention. The function imports plugins if the requsted base class defines a plugin entry point.

Here I was just fixing a typo... but I can of course improve the docstring itself.